gotocompany/assets/v1beta2/asset.proto

```diff
@@ -61,4 +61,8 @@ message Asset {
   // The timestamp when the asset was last modified.
   // This information is expected to be maintained by the system.
   google.protobuf.Timestamp update_time = 102;
+
+  // The timestamp when the asset was last refreshed.
+  // This information is expected to be maintained by the system.
+  google.protobuf.Timestamp refreshed_at = 103;
```

Comment:
IMO, Currently if nothing is changed in the Asset then we don't send an update call to the compass DB, due to the introduction of this flag, even if nothing else changed we will cause an update to the DB. This will result in much more update at the time of asset update/create, is that okay?

Reply:
Of course there are some advantages and drawbacks, but I think that's still okay, since we also reduce the cost of saving the stale assets 🙏  CMIIW mas @haveiss